File: Factorial.java
import java.io.*;
import java.util.*;
class Factorial
{
	public static void main(String args[])throws Exception
	{
		Scanner sc=new Scanner(System.in);
		int n=sc.nextInt();
		int fact=1;
		if(n==0||n==1)
		{
			System.out.println(1);
		}
		else if(n<=20){
			for(int i=2;i<=n;i++)
			{
				fact=fact*i;
			}
			System.out.println(fact);
		}
		else{
			System.out.println("Limit exceeds");
		}
	}
}

File: player/factorial.c
#include<stdio.h>
main()
{
	int n,i,fact=1;
	scanf("%d",&n);
	if(n==0||n==1)
	{
		printf("\n1");
	}
		
	else if(n<=20)
	{
		for(i=2;i<=n;i++)
			fact*=i;
		printf("\n%d",fact);
	}
	else
	{
		printf("\nLimit Exceeds");
	}
		
}
